nntrainer/tensor/tensor_base.cpp

```diff
@@ -29,4 +29,66 @@ size_t TensorBase::getIndex(unsigned int b, unsigned int c, unsigned int h,
   }
 }
 
+void TensorBase::allocateSrcTensor() {
```

Comment:
may be need to consider the ```!src->src_tensor ``` case?

Reply:
fixed! `!src_tensor` case will be handled in `FloatTensor::allocate` and `HalfTensor::allocate`.